src/Views/Component.elm

```diff
@@ -66,80 +73,84 @@ componentView :
-                    [ Component.extractImpacts itemResults
-                        |> Format.formatImpact config.impact
-                    ]
-                , td [ class "pe-3 align-middle text-nowrap" ]
-                    [ button
-                        [ class "btn btn-outline-secondary"
-                        , onClick (config.removeItem component.id)
+                                  else
+                                    text "▼"
+                                ]
+
+                          else
+                            text ""
+                        ]
+                    , td [ class "ps-0 align-middle" ]
+                        [ quantity |> quantityInput config component.id ]
+                    , td [ class "align-middle text-truncate w-100 fw-bold", colspan 2 ]
+                        [ text component.name ]
+                    , td [ class "text-end align-middle text-nowrap" ]
+                        [ Component.extractMass itemResults
+                            |> Format.kg
+                        ]
+                    , td [ class "text-end align-middle text-nowrap" ]
+                        [ Component.extractImpacts itemResults
+                            |> Format.formatImpact config.impact
+                        ]
+                    , td [ class "pe-3 align-middle text-nowrap" ]
+                        [ button
+                            [ class "btn btn-outline-secondary"
+                            , onClick (config.removeItem component.id)
+                            ]
+                            [ Icon.trash ]
                         ]
-                        [ Icon.trash ]
                     ]
                 ]
-          , if not collapsed then
-                tr [ class "fs-7 text-muted" ]
-                    [ th [] []
-                    , th [ class "text-end", scope "col" ] [ text "Quantité" ]
-                    , th [ scope "col" ] [ text "Procédé" ]
-                    , th [ scope "col" ] [ text "Densité" ]
-                    , th [ scope "col" ] [ text "Masse" ]
-                    , th [ scope "col" ] [ text "Impact" ]
-                    , th [ scope "col" ] [ text "" ]
-                    ]
-
-            else
-                text ""
           ]
         , if not collapsed then
-            Component.extractItems itemResults
-                |> List.map2 (processView config.impact) processAmounts
+            List.map2 (elementView config.impact)
+                expandedElements
+                (Component.extractItems itemResults)
 
           else
             []
         ]
 
 
+viewDebug : Component.Results -> Html msg
+viewDebug results =
+    if debug then
+        pre [ class "p-2 bg-light" ]
+            [ results
+                |> Component.encodeResults (Just Definition.Ecs)
+                |> Encode.encode 2
+                |> text
+            ]
+
+    else
+        text ""
```

Comment:
Is it something that you want to keep around for the time being?

Reply:
Frankly, yes. It helped me tremendously for inspecting and debugging the code. I don't think there's a use case for general use hence why it's kept local. But I'd hate having to find this code again to reapply it the next time I need to investigate a computation rendering issue.